platform-emf/src/main/java/com/softicar/platform/emf/editor/EmfAttributeValueLabel.java

```diff
@@ -31,17 +61,69 @@ private void refresh() {
 		setDisplayNone(valueMode.isHidden());
 
 		removeChildren();
-		appendText(getTitle());
+		appendChild(new EmfAttributeValueTitle(attribute.getTitle()));
+		if (showMandatoryIndicator()) {
+			appendChild(new EmfAttributeValueMandatoryIndicator());
+		}
+		attribute//
+			.getHelpDisplayFactory()
+			.map(EmfAttributeValueHelpIndicator::new)
+			.ifPresent(this::appendChild);
 	}
 
-	private IDisplayString getTitle() {
+	private boolean showMandatoryIndicator() {
```

Comment:
IMO, should be `isShowMandatoryIndicator`, otherwise there could be misunderstanding